src/codegen/program.lisp

```diff
@@ -40,6 +40,14 @@
           :else :if (and (typep thing '(cons (member cl:defstruct)))
                          (not (sub-struct-p thing)))
                   :collect thing :into defstructs
+          ;; (DEFCLASS (FOO ... (:INCLUDE ...
+          :if (and (typep thing '(cons (member cl:defclass)))
+                   (not (null (third thing))))
+            :collect thing :into defsubstructs
+          ;; (DEFSTRUCT FOO ...
+          :else :if (and (typep thing '(cons (member cl:defclass)))
```

Comment:
this is matching against DEFCLASS here. Do you mean DEFSTRUCT or are the docs wrong?

Reply:
The comment was wrong